src/main/java/net/sf/rails/game/GameManager.java

```diff
@@ -982,10 +982,11 @@ protected boolean reload(GameAction reloadAction) {
 
         // Check action identity
         int index = 0;
+        int executedCnt = executedActions.size(); // save off the current # of executed actions as it will grow as we execute newly loaded
```

Comment:
I prefer writing out words instead of shortening them:

```
int executedCount = ...
```

Reply:
Agree, except when there are commonly well known abbreviations such as Cnt, Idx AND those abbreviations are secondary words

Reply:
We are talking about 2 letters that you save by using an abbreviation. I can understand when you want to shorten an 8 letter word to 3 letters, but reducing 5 letters to 3 does not help readability in my opinion.

Reply:
PI also vote for the 5Letter version instead of Three :)